frontend/packages/app/public/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/icon.svg" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
Can you add this in the [scaffolding](https://github.com/lyft/clutch/blob/main/tools/scaffolding/templates/gateway/frontend/public/index.html) as well?